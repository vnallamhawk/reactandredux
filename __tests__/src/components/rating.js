import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Rating from '../../../src/components/rating';

describe('Rating Test -- Snapshot',()=>{
    it('capturing snapshot of Rating',()=>{
        const renderer = new ShallowRenderer();
        const result = renderer.render(<Rating/>)
        expect(result).toMatchSnapshot();
    })
})
