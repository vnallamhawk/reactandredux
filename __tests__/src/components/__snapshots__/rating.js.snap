// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating Test -- Snapshot capturing snapshot of Rating 1`] = `
<div>
  <div>
    <ReactStars
      char="★"
      color1="gray"
      color2="#ffd700"
      count={5}
      edit={true}
      half={true}
      onChange={[Function]}
      size={15}
      value={0}
    />
    <div>
       View all 
       reviews  
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-md-6"
    >
      <h3>
         PRO 
      </h3>
      <ReactStars
        char="★"
        color1="gray"
        color2="#ffd700"
        count={5}
        edit={true}
        half={true}
        onChange={[Function]}
        size={15}
        value={0}
      />
      <strong>
         
         
      </strong>
      <p>
         
         
      </p>
    </div>
    <div
      className="col-md-6"
    >
      <h3>
         CON 
      </h3>
      <ReactStars
        char="★"
        color1="gray"
        color2="#ffd700"
        count={5}
        edit={true}
        half={true}
        onChange={[Function]}
        size={15}
        value={0}
      />
      <strong>
         
         
      </strong>
      <p>
         
         
      </p>
    </div>
  </div>
</div>
`;
